testsuite/run_sets/min_proxy.yml

```diff
@@ -8,11 +8,7 @@
 
 # IMMUTABLE ORDER
 
-# initialize Uyuni proxy
-  # one of: proxy_register_as_trad_with_script.feature
-  #         proxy_register_as_minion_with_script.feature
-  #         proxy_register_as_minion_with_gui.feature
-- features/core/proxy_register_as_minion_with_script.feature
-- features/core/proxy_branch_network.feature
+- features/core/proxy.feature
+#- features/core/proxy_branch_network.feature
```

Comment:
It would be good to have a small comment above the commented out line that states why it is commented out.